app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-<h1>users posts</h1>
\ No newline at end of file
+<main>
+    <div class="user-container">
+        <section class="user">
+            <img src=<%= @user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= @user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= @user.posts_counter %> </h5>
+            </div>
+        </section>
+        <section class="user-posts">
+            <h3>All Posts from <%= @user.name %></h3>
+            <% @user.posts.each do |post| %>
+            <a href="/users/<%=@user.id%>/posts/<%=post.id%>" class="user-post-link">
+                <div class="post">
+                    <h4><%= post.title %></h4>
+                    <p><%= truncate(post.text, length: 180) %></p>
+                    <p class="comment-like-count">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+                </div>
+                <div class="comments">
+                    <h4>Comments</h4>
+                    <% post.five_latest_comments.each do |comment| %>
+                        <p  id="single-comment">
+                            <%= comment.author.name %> : <%= comment.text %>
+                        </p>
```

Comment:
- [ ] [OPTIONAL] According to the project requirements, you are suppose to use the five_latest_comments to display only five recent comments, but your app is currently showing more than five comments as shown below. Kindly check and fix it. 

![Screenshot from 2023-06-24 21-13-53](https://github.com/AlaaAlsalem/blogApp/assets/65233860/a7002824-f66f-40e3-a98a-3482504bf99f)